docs/deployment/rds-s3/README.md

```diff
@@ -248,7 +289,62 @@ mysql> select * from observation_logs;
 
 4. Verify the bucket is not empty and was populated by outputs of the experiment.
 
-## Uninstall Kubeflow
+## 5.0 (Optional) Setup credentials configuration for notebooks
+If you are intereseted in having RDS & S3 credentials easily available for your notebooks then you can leverage [PodDefault resources](https://github.com/kubeflow/kubeflow/blob/master/components/admission-webhook/README.md).  
+This is useful when you want to access your RDS & S3 credentials form a notebook using environment variables.  
+For instance, this allows to reference the secrets values without hard coding the values directly in the notebooks.   
+
+### 5.1 Apply the PodDefault
+The `PodDefault` resource is applied to a single namespace therefore we must specify which namespace we want this `PodDefault` to belong to, simply replace `YOUR_NOTEBOOKS_NAMESPACE_HERE` with your notebooks namespace.    
+```shell
+kubectl apply -f ../../../awsconfigs/apps/jupyter-web-app/configs/secrets-pod-default.yaml -n YOUR_NOTEBOOKS_NAMESPACE_HERE
+```
+For instance, if you're using the default `kubeflow-user-example-com` namespace then your command would look like the following :  
+```shell
+kubectl apply -f ../../../awsconfigs/apps/jupyter-web-app/configs/secrets-pod-default.yaml -n kubeflow-user-example-com
+```   
+Note : Since the namespace must exist, this step must be performed **after** installing Kubeflow.  
+
+### 5.2 (Optional) Set PodDefault selected by default in notebook UI  
+The Kubeflow notebook UI allows to select credentials configurations when creating a notebook, by default no configuration is selected but we can add the `PodDefault` label to the list of selected configuration.  
+This results in having the `PodDefault` already selected when creating a new notebook, otherwise the user must select it manually in the notebook UI.  
+  
+We must modify the notebook UI config to the following :  
+```yaml
+  configurations:
+    # List of labels to be selected, these are the labels from PodDefaults
+    value:
+      - add-aws-secret
+```  
+  
+This change can be done before or after installing Kubeflow, here is how to make this modification depending on these 2 options.  
+
+#### 5.2.1 **Option 1: Update notebook UI config before installing Kubeflow**
+Modify the file `awsconfigs/apps/jupyter-web-app/configs/spawner_ui_config.yaml`
```

Comment:
should be call out what modification is needed?

Reply:
The modification to be done is described before. If you asked this question it's probably because it's not clear enough.

I'll duplicate the change to make it clear.

Reply:
Done